indico/modules/payment/blueprint.py

```diff
@@ -30,13 +30,12 @@
                  methods=('GET', 'POST'))
 
 # Event management
-_bp.add_url_rule('/event/<confId>/manage/registration/payment/', 'event_settings', RHPaymentEventSettings)
-_bp.add_url_rule('/event/<confId>/manage/registration/payment/settings', 'event_settings_edit',
-                 RHPaymentEventSettingsEdit, methods=('GET', 'POST'))
-_bp.add_url_rule('/event/<confId>/manage/registration/payment/toggle', 'event_toggle', RHPaymentEventToggle,
-                 methods=('POST',))
-_bp.add_url_rule('/event/<confId>/manage/registration/payment/method/<method>', 'event_plugin_edit',
-                 RHPaymentEventPluginEdit, methods=('GET', 'POST'))
+_bp.add_url_rule('/event/<confId>/manage/payments/', 'event_settings', RHPaymentEventSettings)
```

Comment:
`payments` is more concise in my opinion. You would be managing not **a** payment, not **the** payment, maybe the payment **process**, but that's too long.